src/redux/request.js

```diff
@@ -0,0 +1,166 @@
+import queryString from 'query-string';
+import startsWith from 'lodash/startsWith';
+import { handleActions as reduxHandleActions } from 'redux-actions';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/switchMap';
+import 'rxjs/add/observable/from';
+import 'rxjs/add/observable/of';
+import 'rxjs/add/operator/catch';
+
+// action types
+export const REQUEST_MAKE = '@@ui-eholdings/REQUEST_MAKE';
```

Comment:
What's the convention around the `@@` sigil. I've seen it used for internal redux events.

Reply:
It's basically the agreed-on convention of namespacing actions.

I think since this app is part of another larger ecosystem, we should namespace our actions to prevent our reducers being triggered from other app's poorly-named actions.